### PR TITLE
Issue/7557 document the format used to reference the api itself

### PR DESCRIPTION
**Pull request by Hugo-Inmanta**
# Description

closes https://github.com/inmanta/inmanta-core/issues/7557

~~TODO: When this is approved, open a separate PR for iso6 without the "managed resource" link.~~

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [x] Attached issue to pull request
- [x] Changelog entry
~~- [ ] Type annotations are present~~
~~- [ ] Code is clear and sufficiently documented~~
- [x] No (preventable) type errors (check using make mypy or make mypy-diff)
~~- [ ] Sufficient test cases (reproduces the bug/tests the requested feature)~~
~~- [ ] Correct, in line with design~~
- [x] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )
~~- [ ] If this PR fixes a race condition in the test suite, also push the fix to the relevant stable branche(s) (see [test-fixes](https://internal.inmanta.com/development/core/tasks/build-master.html#test-fixes) for more info)~~

### CODE REVIEW

**Comment by Hugo-Inmanta** [comment]
image

File: docs/reference/apilinks.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
+API self-referencing links
+==========================
+
+Some API endpoints can provide additional information or context by returning links to other API endpoints.
+The following table documents all these links and their format:
+
+
+.. list-table:: Self-referencing links
```
Comment (wouterdb) [inline docs/reference/apilinks.rst:8]: Can we reference this from the api spec itself, where it is used?

**Comment by wouterdb** [comment]
I would keep the full doc in ISO6, only makes it easier

File: docs/reference/apilinks.rst (hunk 1 above)
Comment (Hugo-Inmanta) [reply docs/reference/apilinks.rst:11]: Added a reference to the description of the endpoints (see image below). I tried adding this reference further down the tree, e.g. in the description of the uri of the response object, but I didn't managed to.
image

Comment (wouterdb) [reply docs/reference/apilinks.rst:11]: Good enough I guess.

**Comment by wouterdb** [comment]
I would definitely send out a link to the team to look at it once it is public

**Pull request closed without merging**